internal/backend/crypto/age/ssh.go

```diff
@@ -69,6 +66,31 @@ func (a *Age) getSSHIdentities(ctx context.Context) (map[string]age.Identity, er
 	return ids, nil
 }
 
+func getSSHDir() (string, error) {
+	preferredPath := os.Getenv("GOPASS_SSHDIR")
```

Comment:
Please add details to the config.md and age.md documentation about how this variable works.